reports.js

```diff
@@ -0,0 +1,62 @@
+const axios = require('axios');
+const pulls = require('./pulls.js');
+const updatesUrl = 'http://updates.jenkins.io/plugin-documentation-urls.json';
+const installsUrl = 'https://stats.jenkins.io/jenkins-stats/svg/201910-plugins.csv';
```

Comment:
Is this worth being year-1, month-1?
its just for sorting right?

Reply:
yes, for sorting.